docs/rules/no-restricted-class.md

```diff
@@ -0,0 +1,104 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-restricted-class
+description: disallow specific classes
+since: v7.19.0
```

Comment:
Remove it as it will be added automatically by the release script.

```suggestion
```

Reply:
OK, done. Should I also remove the "This rule was introduced in eslint-plugin-vue v7.19.0." text further down in this file?

Reply:
It will change automatically when you run `npm run update`. However, this is not necessary as it is also done when deploying the document site.